DigitalLearningSolutions.Data/Models/User/CentreAnswersData.cs

```diff
@@ -28,6 +30,7 @@ public class CentreAnswersData
 
         public int CentreId { get; set; }
         public int JobGroupId { get; set; }

```

Comment:
Job group is no longer centre specific so needs moving to the other object MyAccountDetailsData

Reply:
can we remove job group from this model now that we've added it to MyAccountDetailsData? @VladNicol

Reply:
That's still used in some places, it didn't feel like part of the scope of this ticket to remove those. Should I do that?

Reply:
Hmm maybe not, it does seem heavily used in that model. @AlexJacksonDS do you have an opinion?

Reply:
I guess it's probably fine to leave for now until remaining UAR work gets to it. I would expect when UAR is complete, this should be removed, so lets leave a TODO pointing to the cleanup ticket 887 that this is expected to be removed at some point.